.github/workflows/create-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Create GH release
+
+# Create a GitHub release for each new tag.
+# The release notes are taken from the release notes file
+# modified in that commit located in Documentation/RelNotes directory.
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
```

Comment:
Schoudn't we filter out release candidates?

Reply:
This is up to you or other repository owners.
But I think there is no problem publishing release candidates too.

Reply:
Yes.

Reply:
>  But I think there is no problem publishing release candidates too.

I do see a rather big problem publishing them as full releases: https://github.com/git/git/releases/latest would then point to a pre-release, not the latest full release. That's not desirable.